src/status_im2/contexts/communities/discover/events.cljs

```diff
@@ -0,0 +1,45 @@
+(ns status-im2.contexts.communities.discover.events
+  (:require [clojure.set :as set]
+            [clojure.string :as string]
+            [taoensso.timbre :as log]
+            [utils.re-frame :as rf]))
+
+(defn camel->kebab
+  [k]
+  (let [s                  (name k)
+        lower-cased        (string/lower-case
+                            (string/replace s #"([a-z])([A-Z])" "$1-$2"))
+        starts-with-digit? (re-matches #"^\d.*" s)
+        predicate?         (some #(string/starts-with? lower-cased %)
+                                 ["can-" "is-"])]
+    (cond starts-with-digit? s
+          predicate?         (keyword (str lower-cased "?"))
+          :else              (keyword lower-cased))))
+
+(defn rename-keys
```

Comment:
I think the existing file `src/utils/transforms.cljs` is the proper place in the codebase for this sort of thing, since the function should be general enough to not require any specific domain knowledge.